vms/proposervm/acp181/epoch_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package acp181
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/avalanchego/upgrade"
+	"github.com/ava-labs/avalanchego/upgrade/upgradetest"
+
+	statelessblock "github.com/ava-labs/avalanchego/vms/proposervm/block"
+)
+
+func TestEpoch(t *testing.T) {
+	var (
+		now            = time.Now().Truncate(time.Second)
```

Comment:
nit: use hard-coded timestamps to keep tests deterministic
```suggestion
		start            = time.Time{}
```